sample-apps/springboot/src/main/java/com/amazon/sampleapp/FrontendServiceController.java

```diff
@@ -80,11 +80,11 @@ public String healthcheck() {
   // test aws calls instrumentation
   @GetMapping("/aws-sdk-call")
   @ResponseBody
-  public String awssdkCall(@RequestParam(name = "ip", required = false) String ip) {
+  public String awssdkCall(@RequestParam(name = "id", required = false) String id) {
     String bucketName = "e2e-test-bucket-name";
     // Add an (pod) ID to bucketname to associate buckets to specific test runs
-    if (ip != null) {
-      bucketName += "-" + ip;
+    if (id != null) {
+      bucketName += "-" + id;
```

Comment:
IMO this should be testing ID, not pod ID. That way we ensure the ID is distinct to the test run and can be easily found when looking into the telemetry in the account. Update the comment accordingly.

Reply:
Changed to specify testing Id.